tests/integration_tests/utils.py

```diff
@@ -0,0 +1,49 @@
+import logging
```

Comment:
Shall we go with `util.py` for consistency with ~~our other~~`cloudinit.util`?

"Consistency with our other util files":

```
$ find cloudinit -name \*util.py
cloudinit/ssh_util.py
cloudinit/sources/helpers/vmware/imc/guestcust_util.py
cloudinit/tests/test_util.py
cloudinit/util.py
cloudinit/distros/ug_util.py
cloudinit/distros/rhel_util.py
cloudinit/distros/net_util.py
$ find cloudinit -name \*utils.py
cloudinit/temp_utils.py
cloudinit/type_utils.py
cloudinit/cs_utils.py
cloudinit/ec2_utils.py
cloudinit/tests/test_temp_utils.py
cloudinit/distros/bsd_utils.py
```